code/game/machinery/doors/door.dm

```diff
@@ -262,11 +262,6 @@
 	operating = 0
 	air_update_turf(1)
 	update_freelok_sight()
-
-	//I shall not add a check every x ticks if a door has closed over some fire.
-	var/obj/effect/hotspot/fire = locate() in loc

```

Comment:
I left it there intentionally. But, I mean, it doesn't really matter, it doesn't really matter, since LINDA will just melt walls eventually.